magda-web-client/src/helpers/CsvDataLoader.ts

```diff
@@ -111,6 +121,7 @@ class CsvDataLoader {
         this.isLoading = false;
         this.toBeAbort = false;
         this.isPartialData = false;
+        this.skipComplete = false;
```

Comment:
isn't this line not needed because you've already initialized it above?

Reply:
I just want to be safe as `this.skipComplete` should be false at the beginning of the loading process.
The loading process could be triggered more than once for different reasons (e.g. retry line ending)

Reply:
Oh right sorry, I thought this was in the constructor 😬